modyn/config/schema/config.py

```diff
@@ -64,6 +64,9 @@ class DatasetCsvFileWrapperConfig(_DatasetBaseFileWrapperConfig):
     """
 
     separator: str = Field(",", description="The separator used in CSV files.")
+    quote: str = Field("\\0", description="The quote character used in CSV files.")
+    quoted_linebreaks: bool = Field(True, description="Whether linebreaks are quoted in CSV files.")
```

Comment:
Do these default values already fix the behavior for arXiv and huffpost or should we update the modyn example config for them?

Reply:
I only tested it for `huffpost`, there the null-char default (disables quoting more or less). There this resolved all my issues. I can check for arxiv before merging though :)

Reply:
Thanks, that would be great!

Reply:
Training works for `arxiv` :)